miio/integrations/waterheater/viomi/viomiwaterheater.py

```diff
@@ -0,0 +1,271 @@
+            "Error status: {result.error}\n"
+            "Remaining hot water volume: {result.hot_water_volume}%\n"
+            "Device cleaning is required: {result.cleaning_required}\n"
+            "Operation mode type: {result.mode.name} ({result.mode.value}) \n"
+            "Start time in booking mode: {result.booking_time_start}\n"
+            "End time in booking mode: {result.booking_time_end}\n",
+        )
+    )
+    def status(self) -> ViomiWaterHeaterStatus:
+        """Retrieve properties."""
+        properties = SUPPORTED_MODELS.get(
+            self.model, SUPPORTED_MODELS[MODEL_WATERHEATER_E1]
+        )["available_properties"]
+        values = self.get_properties(properties, max_properties=1)
+        return ViomiWaterHeaterStatus(dict(zip(properties, values)))
+
+    @command(default_output=format_output("Powering on"))
+    def on(self):
+        """Power on."""
+        return self.send("set_power", [1])
+
+    @command(default_output=format_output("Powering off"))
+    def off(self):
+        """Power off."""
+        return self.send("set_power", [0])
+
+    @command(
+        click.argument("temperature", type=int),
+        default_output=format_output("Setting target temperature to {temperature}"),
+    )
+    def set_target_temperature(self, temperature: int):
+        """Set target temperature."""
+        min_temp: int
+        max_temp: int
+
+        min_temp, max_temp = SUPPORTED_MODELS.get(
+            self.model, SUPPORTED_MODELS[MODEL_WATERHEATER_E1]
+        )["temperature_range"]
+
+        if not min_temp <= temperature <= max_temp:
+            raise ViomiWaterHeaterException(
+                "Invalid target temperature: %s" % temperature
+            )
+        return self.send("set_temp", [temperature])
+
+    @command(default_output=format_output("Setting bacteriostatic mode on"))
+    def set_bacteriostatic_mode(self):
+        """Set bacteriostatic operational mode (water disinfection mode)."""
+        mode: OperationMode
+        bacteriostatic_mode: bool
+        bacteriostatic_temp: int
+
+        bacteriostatic_mode = SUPPORTED_MODELS.get(
+            self.model, SUPPORTED_MODELS[MODEL_WATERHEATER_E1]
+        )["bacteriostatic_mode"]
+
+        if not bacteriostatic_mode:
+            raise ViomiWaterHeaterException("Bacteriostatic mode is not supported.")
+
+        bacteriostatic_temp = SUPPORTED_MODELS.get(
+            self.model, SUPPORTED_MODELS[MODEL_WATERHEATER_E1]
+        )["bacteriostatic_temperature"]
+
+        mode = OperationMode(self.send("get_prop", ["modeType"])[0])
+
+        # No Bacteriostatic operational mode in Thermostatic mode.
+        if mode == OperationMode.Thermostatic:
+            raise ViomiWaterHeaterException(
+                "Bacteriostatic operational mode is "
+                "not supported in Thermostatic mode."
+            )
+        else:
+            return self.send("set_temp", [bacteriostatic_temp])
```

Comment:
```suggestion

        return self.send("set_temp", [bacteriostatic_temp])
```
No need for else here.